modules/015-admission-policy-engine/hooks/trivy_provider_registry_secrets.go

```diff
@@ -166,15 +168,15 @@ func handleTrivyProviderSecrets(input *go_hook.HookInput, dc dependency.Containe
 	return nil
 }
 
-func dockerConfigByModuleValue(ctx context.Context, cli k8s.Client, value gjson.Result) (*dockerConfig, error) {
+func dockerConfigByModuleValue(ctx context.Context, cli k8s.Client, value gjson.Result) (dockerConfig, error) {
```

Comment:
it will work with pointer
```suggestion
func dockerConfigByModuleValue(ctx context.Context, cli k8s.Client, value gjson.Result) (*dockerConfig, error) {
```